src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -115,41 +142,71 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
                 title: translate('workflowsPage.makeOrTrackPaymentsTitle'),
                 subtitle: translate('workflowsPage.makeOrTrackPaymentsDescription'),
                 onToggle: () => {
-                    // TODO will be done in https://github.com/Expensify/Expensify/issues/368335
+                    const isActive = policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
+                    const newReimbursementChoice = isActive ? CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL : CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
+                    const newReimburserAccountID =
+                        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                        PersonalDetailsUtils.getPersonalDetailByEmail(policy?.reimburserEmail ?? '')?.accountID || policy?.reimburserAccountID || policy?.ownerAccountID;
+                    const newReimburserEmail = PersonalDetailsUtils.getPersonalDetailsByIDs([newReimburserAccountID ?? 0], session?.accountID ?? 0)?.[0]?.login;
+                    Policy.setWorkspaceReimbursement(policy?.id ?? '', newReimbursementChoice, newReimburserAccountID ?? 0, newReimburserEmail ?? '');
                 },
                 subMenuItems: (
-                    <MenuItem
-                        descriptionTextStyle={
-                            isOffline ? StyleUtils.getWorkspaceWorkflowsOfflineDescriptionStyle([styles.textNormal, styles.textSupporting]) : [styles.textNormal, styles.textSupporting]
-                        }
-                        description={translate('workflowsPage.connectBankAccount')}
-                        // onPress={() => Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_CONNECT_BANK_ACCOUNT.getRoute(route.params.policyID))}
-                        // TODO will be done in https://github.com/Expensify/Expensify/issues/368335
-                        shouldShowRightIcon
-                        wrapperStyle={containerStyle}
-                        hoverAndPressStyle={[styles.mr0, styles.br2]}
-                    />
+                    <>
+                        <MenuItem
+                            titleStyle={styles.textLabelSupportingNormal}
```

Comment:
We needed to display the default label in a different font size here. Fixed in https://github.com/Expensify/App/issues/38584